web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -266,7 +250,8 @@ export default function AppVersionHistoryRow(props) {
   const { version, selectedDiffReleases, nothingToCommit,
     isChecked, isNew, renderSourceAndDiff, handleSelectReleasesToDiff,
     yamlErrorsDetails, gitopsEnabled, toggleShowDetailsModal, latestVersion } = props;
-
+  
+  const hideSourceDiff = version?.source.includes("Airgap Install") || version?.source.includes("Airgap Update") || version?.source.includes("Online Install");
```

Comment:
"Airgap Update" should definitely have a diff

Reply:
also, we don't how a diff summary for the initial version today, do we?

Reply:
No we don't, only for updates